src/consumption.cpp

```diff
@@ -824,7 +824,8 @@ ret_val<edible_rating> Character::will_eat( const item &food, bool interactive )
 
     if( food.get_comestible()->parasites > 0 && !food.has_flag( flag_NO_PARASITES ) &&
         !has_flag( json_flag_PARAIMMUNE ) ) {
-        add_consequence( _( "Eating this raw meat probably isn't very healthy." ), PARASITES );
+        add_consequence( string_format( _( "Eating this %s probably isn't very healthy." ), food.tname() ),
```

Comment:
I think "Consuming this %s probably isn't very healthy." would sounds better.

Reply:
I agree.